### PR TITLE
Develop: Unit test for controller added

### PR DESCRIPTION
**Pull request by Rakesh4ever**
Unit test for controller added

### CODE REVIEW

File: src/main/java/com/kumar/entity/User.java [hunk 1]
```diff
@@ -6,16 +6,18 @@
...
+import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 /**
  * @author RakeshKumar created on 19/07/23
  */
 @Entity
+@Builder
 //@Data
```
Comment (Rakesh4ever) [inline src/main/java/com/kumar/entity/User.java:18]: @Data annotation not requiored as we have setter getter

**Review by Rakesh4ever — COMMENTED** [review]
Need to remove commented annotation in model class

**Review by koomar234 — APPROVED** [review]

**Review by Rakesh4ever — COMMENTED** [review]
add latest codebase.

**Review by Rakesh4ever — COMMENTED** [review]
add latest changes

**Pull request merged**